TheBTeam.Web/Controllers/TransactionController.cs

```diff
@@ -45,6 +47,33 @@ public ActionResult UserTransactions()
             return View(model);
         }
 
+        public ActionResult AddTransaction()
+        {
+            return View();
+        }
+
+        // POST: UserController/Create
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult AddTransaction(Transaction model)
+        {
+            try
+            {
+                if (!ModelState.IsValid)
+                {
+                    return View(model);
+                }
+                var id = TempData["id"] as string;
+                var user = _userService.GetById(id);
+                _transactionService.AddTransactionByUser(model, user);
+                return RedirectToAction(nameof(Index));
```

Comment:
i thing that there we should go to users transaction, not all of them

Reply:
It's a little strange but this works well and go to user transaction